language/Types.md

```diff
@@ -152,54 +161,88 @@ since the skolemized type variable ``a`` does not unify with ``Int``.
 
 ## Rows
 
-A row of types represents an unordered collection of named types without duplicates.
+A row of types represents an unordered collection of named types. Duplicate labels are allowed but are distinct.
```

Comment:
I think this is maybe a little unclear; possibly we should use the same text as from the release notes here.